src/Entity/Formation/Axe.php

```diff
@@ -77,20 +77,30 @@ class Axe implements EntityMediaInterface
     private $content;
 
     /**
-     * @var Article[]|Collection
+     * @var Path
      *
-     * @ORM\OneToMany(targetEntity="AppBundle\Entity\Formation\Article", mappedBy="axe")
+     * @ORM\ManyToOne(targetEntity="Path", inversedBy="axes")
+     * @ORM\JoinColumn(nullable=false)
+     *
+     * @Assert\NotBlank
+     */
+    private $path;
+
+    /**
+     * @var Module[]|Collection
+     *
+     * @ORM\OneToMany(targetEntity="Module", mappedBy="axe")
      */
-    private $articles;
+    private $modules;
 
     public function __construct()
     {
-        $this->articles = new ArrayCollection();
+        $this->modules = new ArrayCollection();
     }
 
     public function __toString()
     {
-        return (string) $this->title;
+        return (string) $this->path->getTitle().', '.$this->title;
```

Comment:
we can remove cast here `(string)` as the concatenation operation returns always a string:
`1 . ' hey ' . 2 => string`